presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoPreparedStatement.java

```diff
@@ -67,6 +75,9 @@
         extends PrestoStatement
         implements PreparedStatement
 {
+    private static final Pattern topLevelTypeRegexPattern = Pattern.compile("^(.+?)\\((.+)\\)$");
```

Comment:
We should name these all uppercase since they are constants. The "regex" part can be dropped since it is redundant, so `TOP_LEVEL_TYPE_PATTERN`